src/components/meeting/JitsiComponentWrapper.js

```diff
@@ -2,37 +2,54 @@ import {JitsiComponent} from "e-stella-jitsi";
 import {MeetingFab} from "./MeetingFab";
 import {useState} from "react";
 import {Whiteboard} from "e-stella-whiteboard";
+import {TaskWrapper} from "../task/TaskWrapper";
 
 export const JitsiWrapper = (props) => {
 
     const [jitsiHeight, setJitsiHeight] = useState('calc(100vh - 60px)')
     const [jitsiWidth, setJitsiWidth] = useState('100%')
-    const [whiteboardIsSeen, setWhiteboardIsSeen] = useState(false)
+    const [componentSeen, setComponentSeen] = useState(null)
```

Comment:
Nie wiem czy bym nie zmienił nazwy tego bo jakoś nie czuje nazwy componentSeen ale to może tylko moje wydzimisie XDD

Reply:
Ok, co proponujesz? XD

Reply:
Właśnie największy problem jest że nie wiem tyle tylko że componentSeen dla mnie jest jakoś dziwnie XDD

Reply:
actionComponent? Jakby nie było to to właśnie jest - jakaś akcja (whiteboard, task) to to ma być xDDD